.github/workflows/ci.yaml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+"on":
+  push:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0,12 * * *"
+
+jobs:
+  build:
+    name: Build app
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v3
+
+      # -------- THIS ACTION --------
+      - name: Purge caches
+        uses: ./.
+        with:
+          debug: true
+          # 3 days
+          max-age: 259200
+      # -------- THIS ACTION --------
+
+      # http://man7.org/linux/man-pages/man1/date.1.html
+      - name: Get Date
+        id: get-date
+        run: |
+          echo "date=$(/bin/date -u "+date-%Y-%m-%d-time-%H-%M-%S")" >> $GITHUB_OUTPUT
+        shell: bash
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.npm
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json', 'package.json') }}-${{ steps.get-date.outputs.date }}
+          restore-keys: |
+            ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json', 'package.json') }}-
+      - name: Install packages and build the app
+        run: npm ci
+      - name: Commit & Push changes
+        run: |
+          git config --global user.name github-actions
```

Comment:
Can you use this for the config?

```
git config user.name "$GITHUB_ACTOR"
git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
```

Reply:
done in https://github.com/MyAlbum/purge-cache/pull/2/commits/b6b45dee10f7a4ca2dea29bd46843aca2b34467b

Reply:
By the way, why did you request this change?
With the previous config, automated commits were commited by `github-actions`.
With the new config, the commiter will be the one who triggers the workflow/has modified the workflow file (according to https://github.com/orgs/community/discussions/25067)

Reply:
We are using this config on our internal builds. But maybe it's nicer for public actions to use the way you config worked. You can revert it if you like 🙂

Reply:
Ok :) Done in https://github.com/MyAlbum/purge-cache/pull/2/commits/eb51dbb33299946783edd2caba8a27415d757d2a